src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue

```diff
@@ -235,7 +235,7 @@ export default {
 		generateFile() {
 			this.audioStream.getTracks().forEach(track => track.stop())
 			if (!this.aborted) {
-				this.blob = new Blob(this.chunks, { type: 'audio/mpeg-3' })
+				this.blob = new Blob(this.chunks, { type: 'audio/mpeg; codecs=mp4a.40.2' })
```

Comment:
as far I understand, if the type is audio/mpeg then the extension must be .m4a because then it's a mp4 container with only aac audio in it

if we want to use the ".aac" extension instead, the mimetype would be "audio/aac"

Reply:
This trick is used by both the ios and the android app. But might be it doesn't work with browsers creating such files.

Reply:
[The AudioPreview is shown only when the mimetype of the file starts with `audio`](https://github.com/nextcloud/spreed/blob/2f2b6b9679efd3c7f262499a31f9058b2a7f03c5/src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue#L247), so either the extension needs to be `.m4a` or `.aac` needs to be added to [the server mimetype mappings](https://github.com/nextcloud/server/blob/d47b84be67242960f6af5e8d16ea140e62a0933b/resources/config/mimetypemapping.dist.json).

Reply:
Oops... I did not refresh before writing; it is the same written below :shrug: